contents/docs/surveys/_snippets/ios-surveys-installation.mdx

```diff
@@ -42,7 +42,7 @@ PostHogSDK.shared.setup(config)
 | Multi-question surveys            | ✅                                 |
 | Confirmation message              | ✅                                 |
 | Feedback button presentation      | ❌                                 |
-| Conditional questions             | ❌                                 |
```

Comment:
we need to mention the min. version of the SDK here somewhere
probably in the FAQ too eg https://posthog.com/docs/surveys/troubleshooting#i-set-a-survey-activate-based-on-an-event-why-does-it-show-up-to-users-who-havent-sent-that-event